examples/dataflow-data-generator/data-generator-pipeline/tests/test_get_pyarrow_translated_schema.py

```diff
@@ -0,0 +1,77 @@
+import json
+import logging
+import pyarrow as pa
+import unittest
+from data_generator.ParquetUtil import get_pyarrow_translated_schema
+
+
+class TestPyarrowSchemaTranslator(unittest.TestCase):
+    def test_get_pyarrow_translated_schema(self):
```

Comment:
change request in original PR: https://github.com/GoogleCloudPlatform/professional-services/pull/199#discussion_r262761008